### PR TITLE
feat(entities): add getActiveEntity and getActiveEntities

### PR DESCRIPTION
**Pull request by hrvbernardic**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngneat/elf/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: #185

## What is the new behavior?

Added new get queries for active entities.

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: packages/entities/src/lib/active/active.spec.ts [hunk 1]
```diff
@@ -48,6 +55,37 @@ describe('activeId', () => {
...
+      withActiveId()
+    );
+
+    const store = new Store({ state, config, name: '' });
+
+    expect(store.query(getActiveEntity())).toBeUndefined();
+
+    store.update(addEntities({ id: 1, title: '' }));
+
+    expect(store.query(getActiveEntity())).toBeUndefined();
```
Comment (NetanelBasal) [inline packages/entities/src/lib/active/active.spec.ts:71]: Can you please add tests for `ref` and the return type, please?

File: packages/entities/src/lib/active/active.spec.ts [hunk 2]
```diff
@@ -81,4 +119,45 @@ describe('activeIds', () => {
...
+    expect(store.query(getActiveEntities())).toEqual([]);
+
+    store.update(addEntities({ id: 1, title: '' }));
+
+    expect(store.query(getActiveEntities())).toEqual([]);
+
+    store.update(addActiveIds(1));
+
+    expect(store.query(getActiveEntities())).toEqual([{ id: 1, title: '' }]);
+
```
Comment (NetanelBasal) [inline packages/entities/src/lib/active/active.spec.ts:140]: Can you please add tests for `ref` and the return type, please?

Comment (hrvbernardic) [reply packages/entities/src/lib/active/active.spec.ts:140]: will do, also, I was thinking, would it be a good idea if we could add multiple withActiveId's for different refs (not for this PR)?

Comment (NetanelBasal) [reply packages/entities/src/lib/active/active.spec.ts:140]: Yes, we can do with in other PR.

Comment (hrvbernardic) [reply packages/entities/src/lib/active/active.spec.ts:140]: separate tests for type and ref or?

Comment (hrvbernardic) [reply packages/entities/src/lib/active/active.spec.ts:176]: done

**Pull request merged**